parse_node/parse_class_declaration.ts

```diff
@@ -39,6 +45,14 @@ const getSettersAndGetters = (
       exportText = `export(${typeGodotName ?? "null"}) `
     }
 
+    if (isDecoratedAsExportArgs(setGet)) {
+      exportText = parseExportArgs(setGet, props)
```

Comment:
I am wondering if we should streamline these 3 functions (2+one inline reponse) into one with arguments.

Reply:
I think we should because the same code repeats in two places (in `parse_property_declaration.ts` and `parse_class_declaration.ts`)

Reply:
Sounds good 👍